clang/lib/CodeGen/CGBuiltin.cpp

```diff
@@ -97,10 +97,10 @@ static Value *EmitTargetArchBuiltinExpr(CodeGenFunction *CGF,
   case llvm::Triple::riscv64:
     return CGF->EmitRISCVBuiltinExpr(BuiltinID, E, ReturnValue);
   case llvm::Triple::spirv:
```

Comment:
The change makes it so the spirv target can call `EmitAMDGPUBuiltinExpr` if the triple has AMDHSA. Is that an intended change?

Reply:
nope, I fixed it